klog.go

```diff
@@ -781,24 +781,36 @@ func (l *loggingT) output(s severity, buf *buffer, file string, line int, alsoTo
 		if alsoToStderr || l.alsoToStderr || s >= l.stderrThreshold.get() {
 			os.Stderr.Write(data)
 		}
-		if l.file[s] == nil {
-			if err := l.createFiles(s); err != nil {
-				os.Stderr.Write(data) // Make sure the message appears somewhere.
-				l.exit(err)
+
+		if logging.logFile != "" {
+			if l.file[infoLog] == nil {
+				if err := l.createFiles(s); err != nil {
```

Comment:
should this be `l.createFiles(infoLog)`?
i don't know the details of the duplication issue, but does `logging.logFile` imply `infoLog` severity only?

Reply:
Fixed @neolit123. added some comments to clarify what we are doing